tests/src/org.eclipse.jgit/org.eclipse.jgit/6.5.0.202303070854-r/gradle.properties

```diff
@@ -0,0 +1,2 @@
+library.version = 6.5.0.202303070854-r
```

Comment:
This is a very odd version string. Why does this not look like e.g. `6.5.0`? What is the reason for that?

Reply:
That is version of a third party library and they create release names on that way:
`https://mvnrepository.com/artifact/org.eclipse.jgit/org.eclipse.jgit`

Reply:
I see. All the `org.eclipse.jgit` releases have such odd version names.